projects/plugins/jetpack/package.json

```diff
@@ -177,6 +180,7 @@
 		"commander": "6.2.1",
 		"concurrently": "5.3.0",
 		"config": "3.3.3",
+		"cross-env": "7.0.3",
```

Comment:
Do we really need it? I see a bunch of yarn scripts that passing env variables as part of the script command, and I don't think there were reports of any problems with that. Cc @jeherve for a broader perspective.

Reply:
This was a copy-paste from how the gutenberg fixtures were set up, but as you note there are other instances of env variables passed without this, so have removed for now.